### PR TITLE
added simple validation for outstanding rent

### PR DESCRIPTION
**Pull request by magicmilo**

### CODE REVIEW

File: app/models/case_log.rb [hunk 1]
```diff
@@ -55,6 +55,15 @@ def validate_armed_forces_injured(record)
     end
   end
 
+  def validate_outstanding_rent_amount(record)
+    if record.outstanding_rent_or_charges == "Yes" && record.outstanding_amount.blank?
+      record.errors.add :outstanding_amount, "You must answer the oustanding amout question if you have outstanding rent or charges."
+    end
+    if record.outstanding_rent_or_charges == "No" && !record.outstanding_amount.blank?
```
Comment (baarkerlounger) [inline app/models/case_log.rb]: `.present?` is the opposite of `.blank?` which I think rubocop will want here

Comment (magicmilo) [reply app/models/case_log.rb]: Oops dam cops. Done

**Comment by baarkerlounger** [comment]
Got some merge conflicts that need resolving and need to update API docs but otherwise looks good to me

File: docs/api/DLUHC-CORE-Data.v1.json [hunk 2]
```diff
@@ -1207,6 +1207,17 @@
         ]
       }
     },
-    "securitySchemes": {}
+    "securitySchemes": {
```
Comment (baarkerlounger) [inline docs/api/DLUHC-CORE-Data.v1.json:1210]: Was this intended to be committed or should it have been updating the outstanding amount fields?

**Pull request merged**